DESCRIPTION

```diff
@@ -61,12 +61,10 @@ Suggests:
     xml2
 VignetteBuilder: 
     knitr
-Remotes: 
-    tidymodels/hardhat
 RdMacros: 
     lifecycle
 Config/Needs/website: tidyverse/tidytemplate
 Config/testthat/edition: 3
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.0
```

Comment:
Is there a reason you have dev roxygen2? 7.2.0 just came out!

Reply:
We tend to use the dev version as much as possible.

Reply:
That sounds like a painful way to live 😅